Update unless / else example in conditionals concept

clauddzy opened the pull request:
Tried to clear up the example. Text now matches condition and added the else case.

Discussion is located [here].

[here]: https://forum.exercism.org/t/confusing-unless-else-example-in-conditionals-concept/16293/7

kotp commented on concepts/conditionals/about.md:
```diff
@@ -96,9 +96,11 @@ end
 unless value < 2
-  "1 is not greater than 2"
+  "1 is not less than 2"
+else
+  "1 is less than 2"
```
Because there is an `else` here it strongly suggests that this should be an `if` statement instead, where we swap the conditional statements to be correct.  Avoiding `else` with `unless` is very idiomatic, as it prevents mental gymnastics that otherwise happens.

kotp commented on concepts/conditionals/about.md:
```diff
@@ -96,9 +96,11 @@ end
+  "1 is less than 2"
 end
-# => "1 is greater than 2"
+# => "1 is less than 2"
```
This may be the only change needed.  @meatball133 ??

kotp commented on concepts/conditionals/introduction.md:
```diff
@@ -83,9 +83,11 @@ end
 unless value < 2
-  "1 is not greater than 2"
+  "1 is not less than 2"
+else
+  "1 is less than 2"
```
See the other file where I comment on this.

kotp commented on concepts/conditionals/introduction.md:
```diff
@@ -83,9 +83,11 @@ end
+  "1 is less than 2"
 end
-# => "1 is greater than 2"
+# => "1 is less than 2"
```
This may be the warranted change, in my opinion, as stated earlier.

kotp replied:
As Ellnix expressed, this is more than just a comment it is an expression of evaluation.  This will not end up existing in this form per the discussion on the forums, fortunately.

kotp commented:
Reopened assuming  acceptance of decided "work to be done" in the discusson.

clauddzy commented:
I removed references to `unless` in the **Else statement** paragraph.
I replaced the `unless` example with an `if` example where the `else` branch is executed. That way there are examples for both cases.
I also fixed some other typos I found in the files.

kotp commented on concepts/conditionals/about.md:
```diff
@@ -17,7 +17,7 @@ Here is a list of the operators and an example of when they give a `true` value:
 | Method | Description           | Example |
 | ------ | --------------------- | ------- |
-| <      | less than             | 5 < 4   |
+| <      | less than             | 4 < 5   |
```
Good eyes.

kotp commented on concepts/conditionals/about.md:
```diff
@@ -95,18 +95,20 @@ else
 The `elsif` statement will be executed if the if branch is not executed and the condition of the elsif statement is truthy.
 Elsif statements can be chained together and the first truthy condition will be executed.
-There can also be an else statement at the end of the if statement which will run if non of the earlier statement has been true.
+There can also be an else statement at the end of the if statement which will run if none of the earlier statements have been true.
```
Good here as well.

kotp commented on concepts/conditionals/introduction.md:
````diff
@@ -82,18 +82,20 @@ else
-There can also be an else statement at the end of the if statement which will run if non of the earlier statement has been true.
+There can also be an else statement at the end of the if statement which will run if none of the earlier statements have been true.
 
 ```ruby
````
Can you also attend to this block of code?

The value is one, but the evaluated response states that it is not equal to 1, though it is.

clauddzy replied:
I see what you mean. I will make the changes

kotp replied:
Thank you.

clauddzy replied:
The resulting expression is now mathematically correct.
Another way to go about it would be to simply remove the not in the first part of the condition condition, resulting: `"1 is equal to 1 and 1 is not greater than 2"`. Personally I think the logic in the example I proposed is a bit simpler and lets a beginner focus on the if / elsif statements evaluation, which is the main point of the section.
But I'm fine with either way.

kotp commented:
Thank you for the work @clauddzy!

clauddzy commented:
My pleasure! Yes, this is the first one.
